### ISSUE TITLE
[4.x] Filter articles in admin module mod_latest by workflow stage

### ISSUE THREAD

**Issue by crystalenka** [joomla/joomla-cms#36802]
### Is your feature request related to a problem? Please describe.
You can filter by category or author, but not by workflow stage.

### Describe the solution you'd like
An option to select one (or more) workflow stages to filter articles in the admin module. This would be helpful to quickly see articles that are pending review, for example, since those are not always the most recently published or modified.

### Additional context
It would be ideal if this option only showed up if workflows were enabled.

**Comment by richard67** [comment]
Makes sense to me.

**Comment by akshitrattan** [comment]
I would like to work on this issue!!

**Comment by crystalenka** [comment]
> I would like to work on this issue!!

Anyone can propose a fix in a PR. Give it a try :)

**Comment by niharikamahajan02** [comment]
Does this new option need to be added here?
![image](https://user-images.githubusercontent.com/72392580/150814130-594cc77b-9846-41ee-b4f2-647c0869ef37.png)

**Comment by richard67** [comment]
> Does this new option need to be added here? ![image](https://user-images.githubusercontent.com/72392580/150814130-594cc77b-9846-41ee-b4f2-647c0869ef37.png)

@niharikamahajan02 Yes, and that's here: https://github.com/joomla/joomla-cms/blob/4.1-dev/administrator/modules/mod_latest/mod_latest.xml

**Comment by teekhandekar** [comment]
@Quy Can I work on this ?

**Comment by Quy** [comment]
@teekhandekar In general, it is not necessary to ask for permission. Go for it and thank you for contributing. For this issue, there is already a pull request. Please test #37366. Thanks.

**Comment by vijaybrat** [comment]
I want to work on this issue please  assign me

**Comment by khushiannie** [comment]
hey , I want to solve but don't know how to start .

**Comment by SinghYuvraj0506** [comment]
Hey, I want to work on this issue. Can you assign it to me.

**Comment by brianteeman** [comment]
we do not assign issues

**Comment by charvimehradu** [comment]
Is this issue solved? or am I missing something?
![image](https://github.com/joomla/joomla-cms/assets/121369234/97ce7f29-158a-440e-96ed-ecb891762092)

**Comment by brianteeman** [comment]
this feature request is for the admin module

**Comment by Kartikeya-09** [comment]
@charvimehradu  @Quy  how to open this  dashboard that you shown in your ss.

**Comment by brianteeman** [comment]
@maintainers PLEASE remove these "good first issue" labels

**Comment by charvimehradu** [comment]
Is this issue solved?

**Comment by brianteeman** [comment]
There is a PR for testing https://github.com/joomla/joomla-cms/pull/37366 so this open issue should be closed

**Comment by richard67** [comment]
Closing as having a pull request. Please test #37366 . Thanks in advance.
